lib/flipper/dsl.rb

```diff
@@ -34,6 +34,14 @@ def enabled?(name, *args)
       feature(name).enabled?(*args)
     end
 
+    def enabled_for_some?(name, actor)
+      if enabled?(name)
+        raise Flipper::Error, "The feature #{name.inspect} is enabled for all!"
```

Comment:
It probably doesn't make sense to raise an error. I did so on a whim here, because I think it would be desirable to provide some logging in the case that this condition happens. I think a better solution might be an optional block or argument that can be leveraged by the caller to respond as needed.

Reply:
I went ahead and converted this to a block in https://github.com/jnunemaker/flipper/pull/481/commits/b962c0b574d3f1f6641d659efeb0b0b05ccecf89.